### DISCUSSION TITLE
Drop one stl on top of another

### DISCUSSION THREAD

**Discussion by Manfredo1952** [OrcaSlicer/OrcaSlicer#3612] [Q&A]
I have a model (A), which I cut in two, both A and B are now on the plate.  Now I would like to drop B on top of A.
How can I do that ? When I try to pull it up, it drops down to plate automaticvally.

When I have the second stl outside and am importing, I would import as modifier, which I can lift the keep Z height, but with the oone that I cut and keep on the plate this does not work.

Appreciate ideas- thanks

<img width="570" alt="grafik" src="https://github.com/SoftFever/OrcaSlicer/assets/56330568/3d0b0860-f2c4-4152-8c56-d42d8e11c2bf">

**Comment by janusr** [comment]
Hi, found this the other day tinkering with stacked printing.
Assemble will help you.
https://youtu.be/_zEYix1Ydlk

**Reply by Manfredo1952** [reply]
thanks a lot !   This does help !
